{PROJECT_NAME}/Configurations/Plists/GoogleService/Production/GoogleService-Info.plist

```diff
@@ -2,6 +2,33 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CLIENT_ID</key>
+	<string>394751167018-neam9mprlt90s44eiuvaicaa8r1t4frl.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.394751167018-neam9mprlt90s44eiuvaicaa8r1t4frl</string>
+	<key>API_KEY</key>
+	<string>AIzaSyCpoXIw7nz6ITzby0FLwiQsj4IjzgUmMiE</string>
```

Comment:
is it ok to provide CLIENT_ID, API_KEY in the iOS Template? Should we keep this fields blank for template users?

Reply:
Ok